src/ontology/cl-edit.owl

```diff
@@ -7825,7 +7826,6 @@ SubClassOf(obo:CL_0000564 ObjectSomeValuesFrom(obo:RO_0002202 obo:CL_0000042))
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "FB:ma"^^xsd:string) obo:IAO_0000115 obo:CL_0000565 "A cell found in fat bodies whose primary function is intermediary metabolism."^^xsd:string)
 AnnotationAssertion(oboInOwl:hasOBONamespace obo:CL_0000565 "cell"^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_0000565 "fat body cell"^^xsd:string)
-SubClassOf(obo:CL_0000565 obo:CL_0000181)
```

Comment:
What is the replacement? This makes fat body cell float to the root.

Reply:
@matentzn do we have checks for this?

Reply:
Robot report will warn in case there is a class without a parent..

Reply:
is there a suitable replacement for 'metabolising cell'?

If we do want to obsolete 'metabolising cell', what should be the parent for fat body cell? Native cell?

Reply:
It can't be 'fat cell', since that is mammalian specific per the definition (probably too restrictive).

There's always the wonderful 'stuff accumulating cell', which has 'fat cell' and similar cell types as subclasses.

Reply:
I added 'stuff accumulating cell' as a parent of fat body cell